README.md

```diff
@@ -1,107 +1,215 @@
 # loopback-component-mq
-Loopback Component for working with a Message Queue
+Loopback Component for working with a Rabbit Message Queue.
 
 [![Circle CI](https://circleci.com/gh/fullcube/loopback-component-mq.svg?style=svg)](https://circleci.com/gh/fullcube/loopback-component-mq) [![Dependencies](http://img.shields.io/david/fullcube/loopback-component-mq.svg?style=flat)](https://david-dm.org/fullcube/loopback-component-mq) [![Coverage Status](https://coveralls.io/repos/github/fullcube/loopback-component-mq/badge.svg?branch=master)](https://coveralls.io/github/fullcube/loopback-component-mq?branch=master)
 
-## Installation
+# Overview
+
+This component provides a convenient way to work with a RabbitMQ within a loopback application. This includes:
+
+ - Defining a RabbitMQ topology using the componenet-config.json
+ - Registering message producers and consumers handlers using a mixin.
+ - Inspecting RabbitMQ stats and queue statuses using a RabbitMQ loopback model.
+
```

Comment:
I think we should mention why this package provides both a component and a mixin. Just one or two lines that explains the design choice for having both.